TSChallenge-Work-With-NDVI-and-Met-Data-In-R.Rmd

```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "CHALLENGE: Plot using Facets and Plot Time Series with NDVI data"
```

Comment:
Totally - i just threw this in a doc and gave it a diff name to make sure we saw it was new and different!
Let's create an issue about it not being complete - i'll dig into the lesson next week :)